src/api/schema/topology.rs

```diff
@@ -0,0 +1,65 @@
+use crate::config::{Config, DataType};
+use async_graphql::{Enum, Object};
+use lazy_static::lazy_static;
+use std::sync::{Arc, RwLock};
+
+#[Enum]
+pub enum SourceOutputType {
```

Comment:
Is it the intention that later on our core types such as `Value` will be graphql types? That'd be sweet!

Reply:
Do you mean the TOML `Value` types?

I'm using `MetricValue` in https://github.com/timberio/vector/blob/master/src/api/schema/metrics.rs#L18-L24, which gives you a rough idea how we can 'overlay' existing types and make them 'GraphQL types'.

My intention is to support the full range of config options and have them represented in GraphQL, so I think this will wind up wrapping most of those existing types.